scripts/jenkins/ardana/ansible/roles/ardana_deploy/files/lint-rules/ardana_action_name_should_match_rule.py

```diff
@@ -0,0 +1,52 @@
+#
+# (c) Copyright 2015-2016 Hewlett Packard Enterprise Development LP
+# (c) Copyright 2017 SUSE LLC
```

Comment:
2018

Reply:
same for some other files

Reply:
Sorry Rick, I seem to keep making your life harder at the moment but it's not deliberate, honest ;-)  I originally thought the same as you regarding updating the copyright year, but as a result of the discussion in https://review.openstack.org/#/c/590440/ I realised this is incorrect.  The year should remain as it was first introduced, although it is also acceptable to specify a range where the start year does not change but the end year is updated to match the most recent change to the source.